### ISSUE TITLE
[Feature] Implement ConfigPokemonsToPurge

### ISSUE THREAD

**Issue by Wil32** [NecronomiconCoding/NecroBot#492]
Implement a "blacklist" of pokemon to transfer no matter the IV/CP

ex. Pinsir,Electabuzz,Golbat (those are personal)

**Comment by LasaFawkes** [comment]
I am all up for every additional setting as long as it's optional, but why would you want to do this? Not judging, just genuinely curious :grin:

**Comment by Wil32** [comment]
some Pokemon are not that great even at 100% IV with top CP. 
make room in the inventory.
Some farming spot have pokemon nest which you dont always want beside for the exp they give.

**Comment by LasaFawkes** [comment]
Makes sense. I just don't see myself using it any time soon because I like to min/max every species of pokémon IV and moveset wise, even if they suck :)

**Comment by NecronomiconCoding** [comment]
as this will be clogging up the configs more than it already is, i have moved the enhancement request to a later state

**Comment by salvob41** [comment]
This chain of issues #633, #613, #648 is a neverending loop. 
I reckon the real issue to folllow is #633. Just to be sure and to warn others that might be confused, as I was.
